cli/command/container/signals_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build linux darwin
```

Comment:
Sorry usually this is `!windows`. Any reason to restrict this from other unixes?

Reply:
Ah, no reason! I agree having `signals_windows.go` and  `signals_unix.go` with `!windows` makes sense. For such a small patch it's taking me a while to get it right :)